dev-client/src/navigation/navigators/BottomNavigator.tsx

```diff
@@ -35,7 +34,10 @@ export const BottomNavigator = memo(
       state => state.account.currentUser.data !== null,
     );
 
-    const onHome = useCallback(() => navigation.navigate('HOME'), [navigation]);
+    const onSites = useCallback(
```

Comment:
These changes are already in main and I don't understand why they're showing up in this PR - worth investigating to confirm.

Reply:
I rebased from main. Should be better now.